KR_task3/KR_task3.c

```diff
@@ -0,0 +1,31 @@
+﻿#include <stdio.h>
+#include <stdlib.h>
+#include <locale.h>
+#include <malloc.h>
+
+int main() {
+	setlocale(LC_ALL, ".UTF-8");
+	FILE* file = fopen("test1.txt", "r");
+	if (file == NULL) {
+		printf("Файл не найден.");
+		return 1;
+	}
+	char *data[300] = { 0 };
```

Comment:
```suggestion
	char *data[300] = { NULL };
```
Это ведь массив указателей. Так-то NULL и есть 0, но такая запись делает более явным происходящее